roboteq_driver/src/controller.cpp

```diff
@@ -93,7 +93,6 @@ void Controller::connect() {
 void Controller::read() {
   ROS_DEBUG_STREAM_NAMED("serial", "Bytes waiting: " << serial_->available());
   std::string msg = serial_->readline(max_line_length, eol);
-  static bool receiving_script_messages = false;
```

Comment:
Change this from static var (yikes!) to member var, in preparation for instantiating multiple controller instances in the same binary.